Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in

```diff
@@ -484,6 +484,11 @@ NSURLRequest wrapped by CoreIPCNSURLRequest
 #if USE(AVFOUNDATION)
 AVOutputContext wrapped by CoreIPCAVOutputContext
 #endif
+
+#if USE(PASSKIT) && !HAVE(WK_SECURE_CODING_PKDATECOMPONENTSRANGE)
+PKDateComponentsRange wrapped by CoreIPCPKDateComponentsRange
+#endif
```

Comment:
I think I'm misunderstanding how this works, but wouldn't we want `wrapped by CoreIPCPKDateComponentsRange` if `HAVE(WK_SECURE_CODING_PKDATECOMPONENTSRANGE)`?

And if `!HAVE(WK_SECURE_CODING_PKDATECOMPONENTSRANGE)`, do we not want to `ENCODE_AS_SECURE_CODING(PKDateComponentsRange)`?